package.json

```diff
@@ -158,6 +158,20 @@
           "command": "extension.fabric8AnalyticsWidgetFullStackFromEditor",
           "when": "resourceFilename == go.mod"
         }
+      ],
```

Comment:
Can you add "Dependency Analytic" category to the commands?
https://code.visualstudio.com/api/references/contribution-points#contributes.commands

```
"commands": [
      {
        "command": "fabric8.stackAnalysis",
        "title": "Full Stack Analysis...",
        "category": "Dependency Analytic"
      },
      {
        "command": "extension.fabric8AnalyticsWidgetFullStackFromPieBtn",
        "title": "Open Vulnerability Report",
        "icon": {
          "light": "icon/report-icon.png",
          "dark": "icon/report-icon.png"
        }
      },
```

Also I guess the following commands shall be removed, as it is registered programatically.

```
 {
        "command": "extension.fabric8AnalyticsWidgetFullStackFromExplorer",
        "title": "Dependency Analytics Report..."
      },
      {
        "command": "extension.fabric8AnalyticsStackLogs",
        "title": "Dependency Analytics Logs ..."
      }
```

Reply:
ack

Reply:
@arajkumar I have added the category, but have not removed the commands. those commands don't seem to be registered programmatically.